### PR TITLE
cluster works now

### PR DESCRIPTION
**Pull request by alanren2**

### CODE REVIEW

File: backend/.env [hunk 1]
```diff
@@ -1,3 +1,3 @@
-MONGO_USER=testusr
-MONGO_PASSWORD=password
+MONGO_USER=dbUserAlan
+MONGO_PASSWORD=Nfb1B0OoAK5ObeHP
```
Comment (caiyingshen) [inline backend/.env]: pls take this out and send it to our channel so no one knows your password!!!

Comment (caiyingshen) [reply backend/.env]: oops forgot to request changes

**Review by caiyingshen — CHANGES_REQUESTED** [review]

File: backend/.env [hunk 2]
```diff
@@ -1,3 +1,3 @@
-MONGO_USER=testusr
+MONGO_USER=username
 MONGO_PASSWORD=password
 MONGO_DB=communityconnect-labs
```
Comment (chyku) [inline backend/.env]: could u remove this file and put it in the gitignore + put the contents into channel as a pinned post

**Review by chyku — CHANGES_REQUESTED** [review]
:1st_place_medal:

**Review by chyku — APPROVED** [review]
:fire:

**Review by caiyingshen — APPROVED** [review]
beep beep 🤓

**Pull request merged**